v1/urls.py

```diff
@@ -15,4 +19,27 @@
 urlpatterns = router.urls + [
     path('token/', TokenObtainPairView.as_view(), name='token_obtain_pair'),
     path('token/refresh/', TokenRefreshView.as_view(), name='token_refresh'),
+    path('hello/', hello_world, name='hello-world'),
 ]
+
+schema_view = get_schema_view(
+    openapi.Info(
+        title="Django Workshop API",
+        default_version='v1',
+        description="Django Workshop API created using Swagger for generating documentation",
+        terms_of_service="https://github.com/tilda-center/django-workshop/blob/main/LICENSE",
+        contact=openapi.Contact(email="contact@example.com"),
+        license=openapi.License(name="BSD 2"),
+    ),
+    public=True,
+    permission_classes=(permissions.AllowAny,),
+)
+
+
+# TODO: Modify this as necessary!
+urlpatterns = [
```

Comment:
This variable is defined above, please add items on  lines 43,44 into the existing list.